lib/java/gradle/generateTestThrift.gradle

```diff
@@ -26,6 +26,8 @@ ext.genReuseSrc = file("$buildDir/gen-javareuse")
 ext.genFullCamelSrc = file("$buildDir/gen-fullcamel")
 ext.genOptionTypeJdk8Src = file("$buildDir/gen-option-type-jdk8")
 ext.genUnsafeSrc = file("$buildDir/gen-unsafe")
+ext.genStructOrderTestASrc = file("$buildDir/resources/test/struct-order-test/a")
+ext.genStructOrderTestBSrc = file("$buildDir/resources/test/struct-order-test/b")
```

Comment:
maybe try to use a less nested subdir?

Reply:
Thanks for the feedback. Are you suggesting that instead of:
```
$buildDir/resources/test/struct-order-test/a/
$buildDir/resources/test/struct-order-test/b/
```
it could be improved as:
```
$buildDir/resources/test/struct-order-a/
$buildDir/resources/test/struct-order-b/
```
... or is there another change that would be preferable? thx

Reply:
@Jimexist I guess @nanreh is waiting on any further comments/requests from your side?